src/index.html

```diff
@@ -7,6 +7,308 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+
+      <div class="header__logo"></div>
+
+      <nav class="navigation">
+        <ul class="navigation__items-list">
+          <li class="navigation__items-list__item">
+            <a href="#" class="navigation__link is-active">Apple</a>
+          </li>
+          <li class="navigation__items-list__item">
```

Comment:
According to[ style guide on this project](https://github.com/mate-academy/layout_catalog/blob/master/checklist.md) you have to have an empty line here. Please, fix this for all occurrences in code 
```suggestion
          </li>
          
          <li class="navigation__items-list__item">
```